src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/MetaGuidanceServiceTests.cs

```diff
@@ -0,0 +1,536 @@
+                await statisticsDbContext.AddRangeAsync(releaseVersion1Subject1, releaseVersion2Subject1,
+                    releaseVersion2Subject2);
+                await statisticsDbContext.AddRangeAsync(subject1Filter, subject2Filter);
+                await statisticsDbContext.AddRangeAsync(subject1IndicatorGroup, subject2IndicatorGroup);
+                await statisticsDbContext.AddRangeAsync(subject1Observation1, subject1Observation2,
+                    subject1Observation3, subject2Observation1, subject2Observation2, subject2Observation3);
+                await statisticsDbContext.SaveChangesAsync();
+            }
+
+            await using (var contentDbContext = InMemoryApplicationDbContext(contentDbContextId))
+            await using (var statisticsDbContext = InMemoryStatisticsDbContext(statisticsDbContextId))
+            {
+                var service = SetupMetaGuidanceService(contentDbContext: contentDbContext,
+                    statisticsDbContext: statisticsDbContext);
+
+                // Assert version 1 has one Subject with the correct content
+                var version1Result = await service.Get(contentReleaseVersion1.Id);
+
+                Assert.True(version1Result.IsRight);
+
+                Assert.Equal(contentReleaseVersion1.Id, version1Result.Right.Id);
+                Assert.Equal("Version 1 Release Meta Guidance", version1Result.Right.Content);
+                Assert.Single(version1Result.Right.Subjects);
+
+                Assert.Equal(subject1.Id, version1Result.Right.Subjects[0].Id);
+                Assert.Equal("Version 1 Subject 1 Meta Guidance", version1Result.Right.Subjects[0].Content);
+                Assert.Equal("file1.csv", version1Result.Right.Subjects[0].Filename);
+                Assert.Equal("Subject 1", version1Result.Right.Subjects[0].Name);
+                Assert.Equal("2020_AYQ3", version1Result.Right.Subjects[0].TimePeriods.From);
+                Assert.Equal("2021_AYQ1", version1Result.Right.Subjects[0].TimePeriods.To);
+                Assert.Equal(new List<GeographicLevel>
+                {
+                    GeographicLevel.Country, GeographicLevel.LocalAuthority, GeographicLevel.LocalAuthorityDistrict
+                }, version1Result.Right.Subjects[0].GeographicLevels);
+                Assert.Equal(2, version1Result.Right.Subjects[0].Variables.Count);
+                Assert.Equal("Subject 1 Filter - Hint", version1Result.Right.Subjects[0].Variables[0].Label);
+                Assert.Equal("subject1_filter", version1Result.Right.Subjects[0].Variables[0].Value);
+                Assert.Equal("Subject 1 Indicator", version1Result.Right.Subjects[0].Variables[1].Label);
+                Assert.Equal("subject1_indicator", version1Result.Right.Subjects[0].Variables[1].Value);
+
+                // Assert version 2 has two Subjects with the correct content
+                var version2Result = await service.Get(contentReleaseVersion2.Id);
```

Comment:
Would split out a new test case just for testing amendment, otherwise this test case is testing too many things simultaneously.

Reply:
Cool have done this. The new amendment test case doesn't add the filters, indicator groups, or indicators of the Subjects for brevity.